ADALiOS/ADALiOS/ADLogger+Internal.h

```diff
@@ -19,23 +19,23 @@
 #import "ADLogger.h"
 
 //A simple macro for single-line logging:
-#define AD_LOG(_level, _msg, _code, _info) [ADLogger log:_level message:_msg errorCode:_code info:_info]
+#define AD_LOG(_level, _msg, _code, _info, _correlationId) [ADLogger log:_level message:_msg errorCode:_code info:_info correlationId:_correlationId]
 
 #define FIRST_ARG(ARG,...) ARG
 
 //Allows formatting, e.g. AD_LOG_FORMAT(ADAL_LOG_LEVEL_INFO, "Something", "Check this: %@ and this: %@", this1, this2)
 //If we make this a method, we will lose the warning when the string formatting parameters do not match the actual parameters.
-#define AD_LOG_F(_level, _msg, _code, _fmt, ...) [ADLogger log:_level message:_msg errorCode:_code format:_fmt, ##__VA_ARGS__ ]
+#define AD_LOG_F(_level, _msg, _code, _correlationId, _fmt, ...) [ADLogger log:_level message:_msg errorCode:_code correlationId:_correlationId format:_fmt, ##__VA_ARGS__ ]
 
-#define AD_LOG_ERROR(_message, _code, _info)    AD_LOG(ADAL_LOG_LEVEL_ERROR, _message, _code, _info)
-#define AD_LOG_WARN(_message, _info)            AD_LOG(ADAL_LOG_LEVEL_WARN, _message, AD_ERROR_SUCCEEDED, _info)
-#define AD_LOG_INFO(_message, _info)            AD_LOG(ADAL_LOG_LEVEL_INFO, _message, AD_ERROR_SUCCEEDED, _info)
-#define AD_LOG_VERBOSE(_message, _info)         AD_LOG(ADAL_LOG_LEVEL_VERBOSE, _message, AD_ERROR_SUCCEEDED, _info)
+#define AD_LOG_ERROR(_message, _code, _info, _correlationId)    AD_LOG(ADAL_LOG_LEVEL_ERROR, _message, _code, _info, _correlationId)
```

Comment:
I would prefer if _correlationId was in a consistent place between the format and non-format version of these macros. In the non-format version I would move it to between _code and _info, or between _message and _info